src/Elastic.Apm/Api/Tracer.cs

```diff
@@ -206,5 +206,49 @@ public Task<T> CaptureTransaction<T>(string name, string type, Func<Task<T>> fun
 
 			transaction.End();
 		}, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default);
+
+		public void CaptureError(string message, string culprit, StackFrame[] frames = null, string parentId = null)
+		{
+			var capturedCulprit = string.IsNullOrEmpty(culprit) ? "PublicAPI-CaptureException" : culprit;
```

Comment:
should this check to see if there's a current transaction or current span here, and correlate the error with it?

Reply:
Good catch! 👍  That's also how I documented it :) Fixed now.